### DISCUSSION TITLE
side plate dxf file will not open

### DISCUSSION THREAD

**Discussion by roadgear** [nasa-jpl/open-source-rover#433] [Q&A] [answered]
I am trying to get the body panels cut and find that the side panel dxf file will not open. It will give me an error when I try to use a viewer and when trying to upload to Sculpteo it will also give an error.
What is the solution to this.

**Comment by ericjunkins** [comment]
I can work on getting an MR with the updated file, maybe something got corrupted in the upload :shrug: . If you're itching to get something going soon what you can do is go to the [CAD Model](https://cad.onshape.com/documents/e4f00b1a3d2edb1a84bbba1c/w/8ab8f394324bcc586236ef5d/e/9191e5ad2a70f387b419bc55) and then save your own copy, and then you can open up the body plate part you need and export as DXF and upload that. I just tested and that worked fine to make a dxf

**Reply by roadgear** [reply, marked as answer]
Thanks for the reply, I suspect this would work but the problem is the link takes me to the new version (4.0) of the build. But you know what, as I was poking around in the different versions I did find the file in 3.1.0 that I can open and so have downloaded it and will try sending to get cut.
I'll let you know how I make out.

**Reply by ericjunkins** [reply]
Oh, it was unclear to me you were meaning an older version. I'm not sure what specifically is the issue then, an outdated version of the file isnt working? Where are you getting the file?

**Reply by Achllle** [reply]
Just to make sure, I checked the new version's side plate and found that it works fine.
![image](https://github.com/nasa-jpl/open-source-rover/assets/14060847/30b4f6d2-718f-4ba3-8ab3-5b7a526177db)

**Reply by roadgear** [reply]
the file I downloaded from the 3.1.0 build seems to be working,I was able to upload it to sendcutsend and it is getting made now. 
I had downloaded all the files years ago and only am now getting around to doing the build. I guess it will teach me to check for updated version.

**Reply by roadgear** [reply]
I believe that is the 4.0 version. From the pictures I saw it is quite different from the 3.1.0 version. different rails and other structure pieces and it looks like the body in the new version is also larger.

**Reply by Achllle** [reply]
Yes, when I said 'new version' I was referring to 4.0 (which we've been calling v2). There is no newer version of v3.1.0 ('v1'). The two are not compatible.
